py/selenium/webdriver/common/action_chains.py

```diff
@@ -323,25 +323,54 @@ def send_keys_to_element(self, element, *keys_to_send):
         self.send_keys(*keys_to_send)
         return self
 
-    def scroll(self, x: int, y: int, delta_x: int, delta_y: int, duration: int = 0, origin: Union[str,WebElement] = "viewport"):
+    def scroll_to_element(self, element: WebElement):
```

Comment:
I think we need to stagger this change over two releases? (its a breaking public API change I think), should we keep `scroll(...)` with a `DeprecationWarning`?

Reply:
We can't break APIs like this.

Reply:
This is the downside of doing patch releases instead of waiting for minor releases with new code, but yes, can keep existing method and mark it deprecated instead of deleting it.

Reply:
We can't break APIs like this.

Reply:
Weird... It wasn't intended to be twice

Reply:
I think we posted at the same time and github got confused how to display things.